functions/snapshots/snap_notify_slack_test.py

```diff
@@ -4,431 +4,813 @@
+                    {
+                        'short': False,
+                        'title': 'Link to Finding',
+                        'value': 'https://console.aws.amazon.com/guardduty/home?region=us-east-1#/findings?search=id%3Dsample-id-2'
+                    }
+                ],
+                'text': 'AWS GuardDuty Finding - SAMPLE Unprotected port on EC2 instance i-123123123 is being probed'
+            }
+        ],
+        'channel': 'slack_testing_sandbox',
+        'icon_emoji': ':aws:',
+        'username': 'notify_slack_test'
+    }
+]
+
+snapshots['test_event_get_slack_message_payload_snapshots event_guardduty_finding_low.json'] = [
+    {
+        'attachments': [
+            {
+                'color': '#777777',
+                'fallback': 'GuardDuty Finding: SAMPLE Unprotected port on EC2 instance i-123123123 is being probed',
+                'fields': [
+                    {
+                        'short': False,
+                        'title': 'Description',
+                        'value': '`EC2 instance has an unprotected port which is being probed by a known malicious host.`'
+                    },
+                    {
+                        'short': False,
+                        'title': 'Finding Type',
+                        'value': '`Recon:EC2 PortProbeUnprotectedPort`'
+                    },
+                    {
+                        'short': True,
+                        'title': 'First Seen',
+                        'value': '`2020-01-02T01:02:03Z`'
+                    },
+                    {
+                        'short': True,
+                        'title': 'Last Seen',
+                        'value': '`2020-01-03T01:02:03Z`'
+                    },
+                    {
+                        'short': True,
+                        'title': 'Severity',
+                        'value': '`Low`'
+                    },
+                    {
+                        'short': True,
+                        'title': 'Account ID',
+                        'value': '`123456789`'
+                    },
+                    {
+                        'short': True,
+                        'title': 'Count',
+                        'value': '`1234`'
+                    },
+                    {
+                        'short': False,
+                        'title': 'Link to Finding',
+                        'value': 'https://console.aws.amazon.com/guardduty/home?region=us-east-1#/findings?search=id%3Dsample-id-2'
+                    }
+                ],
+                'text': 'AWS GuardDuty Finding - SAMPLE Unprotected port on EC2 instance i-123123123 is being probed'
+            }
+        ],
+        'channel': 'slack_testing_sandbox',
+        'icon_emoji': ':aws:',
+        'username': 'notify_slack_test'
+    }
+]
+
+snapshots['test_event_get_slack_message_payload_snapshots event_guardduty_finding_medium.json'] = [
     {
-        "attachments": [
```

Comment:
Avoid changing upstream style for non-functional purposes. We want to make it easy for them to accept the patch.

Reply:
Good point. that file was automatically changed when I ran test suite. I will keep the original

Reply:
@jeisen I forgot to run a command `pipenv run format` required to format the code and snapshot to follow the standards